Dockerfile

```diff
@@ -29,14 +29,15 @@ COPY ${REMOTE_SOURCE}/apis ./apis
 COPY ${REMOTE_SOURCE}/controllers ./controllers
 COPY ${REMOTE_SOURCE}/internal ./internal
 
+USER 0
```

Comment:
why we need privileged access in CLO?

Reply:
Build failed making a directory upon change.  I'm open to suggestions.  If this is only in the "builder" image then maybe it's a non-issue as I would expect the "run" base does not use this user.  Do you know?